imap_processing/tests/lo/test_lo_l1b.py

```diff
@@ -465,3 +467,39 @@ def test_convert_tofs_to_eu(attr_mgr_l1b, attr_mgr_l1a):
             expected_tof,
             atol=1e-6,
         )
+
+
+def test_identify_species(attr_mgr_l1b):
+    # Arrange
+    fill_val = attr_mgr_l1b.get_variable_attributes("tof2")["FILLVAL"]
+    l1b_de = xr.Dataset(
+        {
+            "tof2": ("epoch", [14, 80, 500, fill_val]),
+        }
+    )
+
+    expected_species = np.array(["H", "O", "U", "U"])
```

Comment:
Add a value under and between too.
```suggestion
            "tof2": ("epoch", [1, 14, 50, 80, 500, fill_val]),
        }
    )

    expected_species = np.array(["U", "H", "U", "O", "U", "U"])
```